tools/fpgabist/dma/fpga_dma.c

```diff
@@ -738,9 +738,15 @@ static fpga_result poll_interrupt(fpga_dma_handle dma_h) {
 		res = FPGA_EXCEPTION;
 	} else {
 		uint64_t count = 0;
-		read(pfd.fd, &count, sizeof(count));
-		debug_print("Poll success. Return = %d, count = %d\n",poll_res, (int)count);
-		res = FPGA_OK;
+		ssize_t bytes_read = read(pfd.fd, &count, sizeof(count));
+		if(bytes_read <= 0) {
+			fprintf( stderr, "Error: %s\n",
+				bytes_read < 0 ? strerror(errno) : "zero bytes read");
```

Comment:
Should we provide a little more context in the error message? Maybe throw in something about what operation was being performed?

Reply:
@r-rojo that makes sense.  I added error check to deal with the warning triggered by not checking return value of read, but don't really expect it to be triggered.  But if it ever is, whoever if debugging the problem might appreciate having more information.

Reply:
👍